integration_tests/Dockerfile-wsgi-patch

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9-slim-bullseye
+
+RUN apt update && apt install --assume-yes gcc
+
+COPY integration_tests/assets/bin/simple-wsgi-server.py /usr/local/bin
+COPY . /usr/local/src/xivo-lib-python
+
+RUN cd /usr/local/src/xivo-lib-python && \
+    python3 -m pip install -r requirements.txt && \
+    python3 -m pip install -r integration_tests/test-requirements.txt && \
```

Comment:
I would not install `test-requirements.txt`. You only need the specific requirements of `cheroot`(already present in requirements.txt`), `flask`(also already present in requirements.txt) and `flask-restful`. The other requirements of `test-requirements.txt` are superfluous.

You can either just install the additional requirements(`flask-restful`) directly, or have a separate `wsgi-patch-requirements.txt`.

Reply:
I wasn't sure which direction to go concerning the `requirements.txt` files, I'll just use the main one and install the last missing dependency in the Dockerfile

Reply:
Updated